examples/kafka-authentication-sasl-plain-producer/kafka_authentication_sasl_plain_producer.bal

```diff
@@ -9,25 +9,25 @@ kafka:AuthenticationConfiguration authConfig = {
     // Check Ballerina `config` APIs to see how to use encrypted values instead of plain text values here.
```

Comment:
```suggestion
    // Check Ballerina `config` APIs to see how to use encrypted values instead of plain text values here.
```
Is this an instruction for the user?

Reply:
yes

Reply:
@dilanSachi IINM, this instruction is outdated since the `config` API is removed now. Could you check?

Reply:
Changed appropriately